presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java

```diff
@@ -830,6 +830,15 @@ public void testStringFunctions()
 
         // Reverse
         assertQuery("SELECT comment, reverse(comment) FROM orders");
+
+        // Normalize
+        assertQuery("SELECT normalize(null)");
```

Comment:
Thanks @pdabre12 

All the queries from 835 - 839 will be executed at the co-ordinator as they use constants. Please add more queries using columns like those on line 840/841. But orders table is very large. Maybe use a field from nations or regions table instead. Or another option is to create a table and insert interesting data into it in the test itself.